docker/conf/docker-compose.github-actions.yml.dist

```diff
@@ -32,8 +32,12 @@ services:
         restart: always
         image: storefront-image
         container_name: shopsys-framework-storefront
+        command:
+            - dev
```

Comment:
is the command here necessary? If I understand it correctly, this docker-compose file is used only for standards checking, or? So the command can be the default?

Reply:
Container would stop after running `up` as no running process would be present